resources/public/api/conf/2.0/update_accounting_type.yaml

```diff
@@ -0,0 +1,110 @@
+summary: Update Accounting Type
+description: |
+  This endpoint allows a customer to update the Accounting Type that they have on record. This can be done at the end of the year after their Final Declaration.
+
+  ### Test data
+  Scenario simulations using <code>Gov-Test-Scenario</code> headers is only available in the sandbox environment.
+  
+  | Header Value (Gov-Test-Scenario)         | Scenario                                                                                                                                      |
+  |------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
+  | N/A - DEFAULT                            | Simulate a successful response with a self-employment business. <br/> Use the following businessId for this scenario: <br/> - XBIS12345678901 |
+  | NOT_FOUND                                | Simulate a scenario where no data is found.                                                                                                   |
+  | OUTSIDE_AMENDMENT_WINDOW                 | Simulates the scenario where request cannot be completed as it is outside the amendment window.                                               |
+  | STATEFUL                                 | Performs a stateful retrieve.                                                                                                                 |
+
+
+
+
+parameters:
+  - $ref: './common/pathParameters.yaml#/components/parameters/nino'
+  - $ref: './common/pathParameters.yaml#/components/parameters/businessId'
+  - $ref: './common/pathParameters.yaml#/components/parameters/taxYear'
+  - $ref: './common/headers.yaml#/components/parameters/acceptJson'
+  - $ref: './common/headers.yaml#/components/parameters/authorization'
+  - $ref: './common/headers.yaml#/components/parameters/testScenarioHeader'
+
+tags:
+  - Accounting Type
+
+security:
+  - User-Restricted:
+      - read:self-assessment
+
+requestBody:
+  content:
+    application/json:
+      schema:
+        $ref: './schemas/updateAccountingType/request.json'
+      examples:
+        example-request:
+          description: Example request given any tax year
+          value:
+            $ref: './schemas/updateAccountingType/request.json'
```

Comment:
This needs to be an example and not the original schema.  If you leave it like this the example request renders like this:

![Screenshot 2025-06-06 at 12 10 55](https://github.com/user-attachments/assets/b84a4b0c-5993-4e69-9b5a-d44d619a3686)